### DISCUSSION TITLE
I can´t add Data Base in SQLiteStudio

### DISCUSSION THREAD

**Discussion by beatrisu21** [pawelsalawa/sqlitestudio#4905] [General]
<img width="1434" alt="Screenshot 2023-12-10 at 01 30 30" src="https://github.com/pawelsalawa/sqlitestudio/assets/142795415/0d2c518f-7a10-4bfe-a08a-1f24a0b61d57">

**Operating system**
MacOS Sonoma 14.1.2

**SQLiteStudio version**
SQLiteStudio 3.4.4

**Comment by pawelsalawa** [comment]
Is it existing database, or are you trying to create a new one?

**Reply by beatrisu21** [reply]
There I was trying to create a new one. Then  I created a db by myself and added it into SQLiteStudio. I got a similar message trying to create a table.
<img width="1434" alt="Screenshot 2023-12-10 at 14 17 23" src="https://github.com/pawelsalawa/sqlitestudio/assets/142795415/a630bf08-a3a8-4871-adf6-af8cb272267a">

**Reply by beatrisu21** [reply]
I even tried to grant full access to the disk. It is still the same

**Reply by beatrisu21** [reply]
Permission string for db.sqilte:
-rw-r--r--  1 User  staff        0 Dec 10 01:43 db.sqlite

I'm so desperate :(

**Reply by pawelsalawa** [reply]
On the screenshot above you did add database correctly. The problem is, that you did not open the database prior to creating a table. Go to "View" menu and enable "Database list" there. Then double click on the database to open it.

**Comment by jvaidya87** [comment]
Hi @pawelsalawa - I appreciate all of your work keeping sqlitestudio up to date.

I have a few students using SQLiteStudio version 3.4.4 on the latest MacOS and they are also receiving an error message stating that they cannot open a DB file when they try to use the navigation menu to create a new DB.

Do you think this is a permissions issue? If they were to open up the cli to generate a new DB, then open it via the GUI and populate it in that manner, do you think that would work?

**Reply by pawelsalawa** [reply]
Well, have you tried doing it together with them? To make sure what they do correct?

It is possible that MacOS is blocking file access to SQLiteStudio due to some reasons. For example:
- user tries to create the database file in a folder, to which he/she has no (write) access to
- MacOS blocks file access for SQLiteStudio entirely, because it treats unsigned application as a threat. It was not the case so far (at least couple months ago, when I tested it on MacOS then) - for this one I have no answer, you can read more about it at https://github.com/pawelsalawa/sqlitestudio/issues/3932
